### ISSUE TITLE
Add donation buttons to README

### ISSUE THREAD

**Issue by idpaterson** [idpaterson/alfred-wunderlist-workflow#9]
The Accessibility Inspector provided in OS X offers some very basic insight into the structure of the UI in an OS X app like Wunderlist, but it is cumbersome and lacking a lot of features compared to [UI Browser](http://pfiddlesoft.com/uibrowser/index.html). Unfortunately UI Browser has a $55 license fee once the free trial, which helped significantly in the development of this workflow, expires. 

Donations could be helpful in funding the cost of that license. Add a section at the bottom of the README to accept donations and explain how they will be used.
### Donation platforms
- [Flattr](https://flattr.com/embed)
  - Easy, but involves foreign transaction fees if I were to actually withdraw money
  - Would probably just use any donations received to flattr other projects
- [Dwolla](http://developers.dwolla.com/bd/button)
  - Zero transaction fees 
  - US only
- PayPal, Google Wallet, etc
  - _meh…_
  - Higher expenses plus the air of desperation
